backend/controllers/SiteController.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace backend\controllers;
 
-use Yii;
+use yii;
```

Comment:
Problem synopsis
      Namespace reference is not case-sensitive

**yiisoft/yii2/Yii.php**
```
namespace yii;
//require(__DIR__ . '/BaseYii.php');
...
class Yii extends BaseYii

```
**use yii\Yii;**